pkg/tfvars/baremetal/baremetal.go

```diff
@@ -156,8 +156,7 @@ func TFVars(numControlPlaneReplicas int64, libvirtURI, apiVIP, imageCacheIP, boo
 
 		// Properties
 		propertiesMap := map[string]interface{}{
-			"local_gb":     profile.LocalGB,
-			"cpu_arch":     profile.CPUArch,
+			"cpu_arch":     "x86_64",
```

Comment:
I this a step backwards? I couldn't find in install-config where arch is so came to the conclusion its not supported for day1 ?

Reply:
Even better, you can use `bmh.Spec.Architecture` (and omit the value if it's empty).